src/providers/twitch/TwitchBadges.hpp

```diff
@@ -5,8 +5,14 @@
 #include <unordered_map>
 
 #include "common/UniqueAccess.hpp"
+#include "messages/Image.hpp"
+#include "util/DisplayBadge.hpp"
 #include "util/QStringHash.hpp"
 
+#include "pajlada/signals/signal.hpp"
+
+#include <shared_mutex>
```

Comment:
Why is `<unordered_map>` all the way at the top?

Reply:
We don't have a standard of include order, so I'll mark this resolved for now.